### DISCUSSION TITLE
Switching languages on Github?

### DISCUSSION THREAD

**Discussion by Long-Web-End** [community/community#136913] [Other Feature Feedback, Questions, & Ideas] [answered]
### Select Topic Area

Question

### Body

Hello!

I'd like to know if is possible to change Github's interface language. I can't find it in my account Settings. Thank you.

**Comment by agustrodmar** [comment, marked as answer]
Hi @Long-Web-End 

Despite it was possible back in the past, I'm afraid that's not longer possible. Github does not support other languages apart from English. You can find this info in older posts like #62798 Have a nice day.

**Comment by MichaelDzaychiks** [comment]
hello

As of November 18, 2016, GitHub ceased support for languages other than English.
